### DISCUSSION TITLE
How can I set a default project and activity and hide the two fields ?

### DISCUSSION THREAD

**Discussion by Crovitche-1623** [kimai/kimai#5095] [Questions & Answers] [answered]
I saw in similar discussions that I could set a default project and a default activity (#4721) using CustomCSS.

Could someone provide a complete example because that would be a prerequisite for my company to use this application.

**Comment by kevinpapst** [comment]
Its custom Javascript (not CSS) and all in the documentation.

**Reply by Crovitche-1623** [reply]
@kevinpapst, Thanks for the quick answer. I guess this is in this section: https://www.kimai.org/documentation/plugin-custom-content.html ?

**Reply by kevinpapst** [reply, marked as answer]
Yes. Search for "Remove the activity from the timesheet form. You have to create one global activity and grab the ID from the URL"

**Reply by Crovitche-1623** [reply]
Thanks
